arrows/ffmpeg/ffmpeg_init.cxx

```diff
@@ -35,22 +35,70 @@
 
 #include "ffmpeg_init.h"
 
+
+#include <algorithm>
+#include <cctype>
 #include <mutex>
 
 extern "C" {
 #include <libavformat/avformat.h>
 }
 
+#include <vital/logger/logger.h>
+
 std::mutex ffmpeg_init_mutex;
 
-//--------------------------------------------------------------------------------
+static auto ffmpeg_logger = kwiver::vital::get_logger( "arrows.ffmpeg" );
+
+//-----------------------------------------------------------------------------
+// trim whitespace from end of string (in place)
+static inline void rtrim(std::string &s) {
+    s.erase(std::find_if(s.rbegin(), s.rend(),
```

Comment:
There are string trimming functions in vital/util/string.h

Reply:
okay, I'll use that